theme.qml

```diff
@@ -309,12 +333,20 @@ FocusScope {
             settings_games_layout: "Mise en page des jeux",
             settings_games_gridItemsPerRow: "Grille - nombre de jeux par ligne",
             settings_games_gridItemsViewableRows: "Grille - nombre de lignes",
-            menu_settings: "réglages",
-            menu_home: "accueil",
-            menu_collections: "collections",
-            menu_games:"jeux"
+            menu_settings: "Réglages",
+            menu_home: "Accueil",
+            menu_collections: "Collections",
+            menu_games:"Jeux"
         },             
         "pt": {
+            global_dimm: "dimm",
+            global_bright: "bright",
+            global_Dark: "Dark",
+            global_Light: "Light",
+            global_OzoneDark: "OzoneDark",
+            global_SteamOS: "SteamOS",
+            global_mono: "mono",
+            global_color: "color",

```

Comment:
```suggestion
            settings_collection_accentColor_dimm: "dimm",
            settings_collection_accentColor_bright: "bright",
            settings_general_colorScheme_Dark: "Dark",
            settings_general_colorScheme_Light: "Light",
            settings_general_colorScheme_OzoneDark: "OzoneDark",
            settings_general_colorScheme_SteamOS: "SteamOS",
            settings_general_logoVariant_mono: "mono",
            settings_general_logoVariant_color: "color",
```
global is for text strings which can appear in many places in the theme. I assume these strings will never be needed outside the settings screen. Therefore I would have adapted the naming scheme to this one

Reply:
ok good information